src/structures/User.js

```diff
@@ -186,6 +188,7 @@ class User extends Base {
    * Otherwise a link to their default avatar will be returned.
    * @param {ImageURLOptions} [options={}] Options for the Image URL
    * @returns {string}
+   * @deprecated Use avatarURL() with includeDefault set to true instead.
```

Comment:
```suggestion
   * @deprecated
```
consistency with other `@deprecated` tags